flink-table/flink-table-api-java/src/test/java/org/apache/flink/table/functions/FunctionDefinitionUtilTest.java

```diff
@@ -129,4 +174,14 @@ public TypeInformation getAccumulatorType() {
 			return TypeInformation.of(Object.class);
 		}
 	}
+
+	/**
+	 * Test function.
+	 */
+	public static class TestTableAggFunctionWithoutResultType extends TableAggregateFunction<Long, Long> {
+		@Override
```

Comment:
Can we extend the `TestTableAggFunction` instead so that we can reuse the code ?

Reply:
No, reuse makes code strange. Better to distinguish them clearly.